### DISCUSSION TITLE
JSON files not found when deploying to vercel

### DISCUSSION THREAD

**Discussion by adamsulc98** [amannn/next-intl#1257] [Q&A]
### Description

On localhost, everything works as it is supposed to, but when I deploy the app to vercel, it seems that app cannot find the json files.
The build itself doesnt throw any error, but In console there is this:

```
    Failed to load resource: the server responded with a status of 404 ()
    _next/data/XoP-CXuDxDooprFNUoha3/cs.json
```

So it seems that vercel cannot find json files with translations. The json files are stored in **messages** folder that is on root.

This is my i18n configuration:

```
    export default getRequestConfig(async ({ locale }) => {
      if (!locales.includes(locale)) notFound();
      return {
        messages: (await import(`../../../messages/${locale}.json`)).default,
      };
    });
```

My middleware:

```
    export default createMiddleware({
      locales,
      defaultLocale,
      localePrefix,
      pathnames,
    });
    
    export const config = {
      matcher: ['/', '/(cs|en)/:path*'],
    };
```

next.config.mjs:

```
    const withNextIntl = createNextIntlPlugin(
      './src/business-web-core/src/i18n.ts',
    );
    
    /** @type {import('next').NextConfig} */
    const nextConfig = {};
    
    export default withNextIntl(nextConfig);
```

And navigation:

```
    export const { Link, getPathname, redirect, usePathname, useRouter } = createLocalizedPathnamesNavigation({
      locales,
      pathnames,
      localePrefix,
    });
```

Thank you in advance.

### Verifications

- [X] I've verified that the problem I'm experiencing isn't covered in [the docs](https://next-intl-docs.vercel.app/).
- [X] I've searched for similar, existing issues on GitHub and [Stack Overflow](https://stackoverflow.com/search?q=next-intl).
- [X] I've compared my app to [a working example](https://next-intl-docs.vercel.app/examples) to look for differences.

### Mandatory reproduction URL

https://personal-web-six-roan.vercel.app/cs

### Reproduction description

Open console and see the 404 error.

### Expected behaviour

The app should be able to load translation files and display pages.

**Comment by amannn** [comment]
I unfortunately don't see an obvious problem with the code you've provided, sorry :/. Since this works locally, it seems like something is off with the Vercel deployment.

I'll move this to a discussion since this is a usage question and not a bug report.
